submissions/nwokejeme-immaculeta/dom/signup.js

```diff
@@ -0,0 +1,178 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    otherName = document.getElementById('otherName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirmPassword = document.getElementById('confirmPassword');
+
+    function required(field, event) {
+        if (field.value === '') {
+            field.nextElementSibling.innerHTML = "Please enter your Full Name";
+            event.preventDefault();
```

Comment:
You are repeating yourself

Kindly pass in the error message as a parameter in the function